components/service/fretboard/src/main/java/mozilla/components/service/fretboard/JSONExtensions.kt

```diff
@@ -0,0 +1,62 @@
+package mozilla.components.service.fretboard
+
+import org.json.JSONArray
+import org.json.JSONObject
+import java.util.TreeMap
+
+@Suppress("UNCHECKED_CAST")
+internal fun <T> JSONArray?.toList(): List<T> {
+    if (this != null) {
+        val result = ArrayList<T>()
+        for (i in 0 until length())
+            result.add(get(i) as T)
+        return result
+    }
+    return listOf()
+}
+
```

Comment:
Is this file still needed? Didn't we move them into ktx? Or are there new ones in here?

Reply:
No, actually it isn't needed. I have updated the pull request to use the extensions from ktx instead